WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -106,18 +106,32 @@ class StoreOnboardingViewModel: ObservableObject {
 
 private extension StoreOnboardingViewModel {
     @MainActor
-    private func loadTasks() async throws -> [StoreOnboardingTaskViewModel] {
-        try await withCheckedThrowingContinuation { continuation in
+    func loadTasks() async throws -> [StoreOnboardingTaskViewModel] {
+        let shouldManuallyAppendLaunchStoreTask = await isFreeTrialPlan()
+        let tasksFromServer: [StoreOnboardingTask] = try await withCheckedThrowingContinuation({ continuation in
```

Comment:
nit: how about also moving this remote tasks fetching task to a separate `async throws` function so that it's easier to read how the two async results are used?

Reply:
Done in ec1fa35bdce98294cffa38e433fcb84177491152